ghastcoiler/minions/rank_2.py

```diff
@@ -86,7 +86,11 @@ def __init__(self, **kwargs):
                          **kwargs)
 
     def gives_attack_defense_bonus(self, other_minion):
-        return 2, 0
+        if other_minion.types is MinionType.Murloc:
```

Comment:
other_minion.types is a list of types (for example for the hero that makes the Amalgam or whatever it is called) so you should check for inclusion like this:

if MinionType.Murloc is in other_minion.types:
    ...